src/modules/Providers/ProviderRow.tsx

```diff
@@ -70,7 +70,7 @@ const TextWithIcon = ({ value, Icon }: { value: string; Icon: JSXElementConstruc
 const ProviderLink = ({ value, entity, t }: CellProps) => (
   <>
     <ResourceLink kind={entity.kind} name={value} namespace={entity?.namespace} />{' '}
-    {!entity.url && (
+    {entity.type === 'openshift' && !entity.url && (
```

Comment:
If we touch host provider detection logic then it makes also sense to use the same approach for [new kebab actions](https://github.com/kubev2v/forklift-console-plugin/blob/e84ab1d587ffd3b7ed8c57c3a325afa84c5d1e28/src/modules/Providers/providerActions.tsx#L26)  i.e. via a common helper.